app/api.py

```diff
@@ -0,0 +1,16 @@
+import psycopg2 
+import psycopg2.extras
+from app.main import DATABASE_URL, app
+
+
+
+@app.route("/api/parties")
+def parties():
+    db_conn = psycopg2.connect(DATABASE_URL)
```

Comment:
We should probably talk about moving this out to a connection pool at some point.

Reply:
basically at the moment we are making a connnection to the database every time a request is made. This takes time, and that time results in a slow web page for the user.

Instead what we can do is create a "pool" of database connections that are shared between all the requests that being made to the server. when a function making a request needs a database connection they can take one from the pool. when they are done making queries instead of closing the connection, the connection is returned to the pool to be used by the next request.

This way you avoid the overhead of starting a new connection on each request.